src/content/structured/get-started/react.mdx

```diff
@@ -136,3 +136,19 @@ Replace any instances of `<svg>` with `<SlottedSVG>`.
   </SlottedSVG>
 </IcTopNavigation>
 ```
+
```

Comment:
I think we need to add something about how to scale icons in the `<SlottedSVG>` component as we recently got a community question on this. 

This is the sentence I wrote in a previous comment: "To scale the icons, use the [`viewBox` attribute](https://css-tricks.com/scale-svg/#aa-the-viewbox-attribute)." Not sure if it fits better on line 111 or in the MDI section. Probably best on line 111 but we'd need to show the viewBox attribute being used in the example above

Reply:
@GCHQ-Developer-465 small thing but please can you add this link: https://css-tricks.com/scale-svg/#aa-the-viewbox-attributeon the words, "viewBox attribute"?